sql/auth/sql_user.cc

```diff
@@ -1768,13 +1778,25 @@ bool set_and_validate_user_attributes(
                              std::string(Str->host.str), gen_password, 1};
       generated_passwords.push_back(p);
     }
+#ifdef WITH_WSREP
+    if (auth->generate_authentication_string(outbuf, &buflen, inbuf,
+                                             inbuflen) ||
+        (verify_passwd_history &&
+         auth_verify_password_history(thd, &Str->user, &Str->host,
```

Comment:
There is another call to auth_verify_password_history() in this function. Should not we check verify_passwd_history flag value in its case as well?